sqlalchemy-stubs/sql/functions.pyi

```diff
@@ -1,13 +1,17 @@
-from typing import Any, Optional
+from typing import Any, Optional, TypeVar, Generic
 from .base import Executable as Executable
 from .elements import ColumnElement as ColumnElement
 from .selectable import FromClause as FromClause
 from . import util as sqlutil
+from . import sqltypes
+from .schema import Sequence
 from .visitors import VisitableType as VisitableType
 
+_T = TypeVar('_T')
+
 def register_function(identifier, fn, package: str = ...): ...
 
-class FunctionElement(Executable, ColumnElement, FromClause):
+class FunctionElement(Executable, ColumnElement[_T], FromClause, Generic[_T]):  # type: ignore
```

Comment:
This `# type: ignore` looks strange, what is the cause?

Reply:
I'll add a comment